tools/cldr-code/src/main/java/org/unicode/cldr/util/StandardCodes.java

```diff
@@ -1076,7 +1076,7 @@ private void add(CodeType type, String code, List<String> otherData) {
     static final String registryName = CldrUtility.getProperty("registry", "language-subtag-registry");
 
     public enum LstrType {
-        language("und", "zxx", "mul", "mis", "root"),
+        language(CLDRLocale.UNKNOWN_LOCALE_NAME, "zxx", "mul", "mis", CLDRLocale.ROOT_NAME),
```

Comment:
I'm a little, or more than a little, concerned about this. CLDRLocale has static init implications.  it might be better to move these string constants under `CLDRUtility` or even `CLDRFile`, for that reason.   Otherwise it will bring StandardCodes into the static init issue.